src/main/resources/db/migration/V9__CREATE_V2_BEHOV_TABLE.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE IF NOT EXISTS V2_BEHOV
```

Comment:
@tonjemjovik @atleno @androa Vi flatet ut parametrene i ny versjon av behov tabell, istedet for å serialisere Behov til jsonb - Jeg tror det kan være lurt for senere bruk. Noen tanker?

Reply:
Jeg har god erfaring med å gjøre begge deler. Flate ut de vi trenger til oppslag, men ta vare på hele greia. Det gjør det så mye enklere å migrere framover når vi legger til nye felter (vi kan begynne å produsere de før vi (vet at vi trenger) å gjøre schema endringer.

Reply:
god idé, altså ha `data` jsonb felt?